website/src/Footer.js

```diff
@@ -0,0 +1,14 @@
+import m from 'mithril'
+import './styles/footer.scss'
+
+export const Footer = {
+  view: vnode =>
+    m('footer', {}, [
+      m('.text', 'More information at '),
+      m(
+        'a.link',
+        { href: 'https://www.quaternion.media/alfred/' },
+        'quaternion.media/alfred'
+      ),
+    ]),
+}
```

Comment:
I simplified this component from a `function()` -> `const {}`.

This is the same principle as [simplifying the `Layout` component in the last PR](https://github.com/quaternionmedia/alfred/pull/75#discussion_r934030154).